src/main/scala/temple/generate/language/service/go/GoServiceGenerator.scala

```diff
@@ -96,22 +106,71 @@ object GoServiceGenerator extends ServiceGenerator {
     sb.toString
   }
 
+  private def generateJsonMiddleware(): String =
+    readFile("src/main/scala/temple/generate/language/service/go/genFiles/JsonMiddleware.go")
+
+  private def generateHandler(serviceName: String, endpoint: Endpoint): String =
+    s"func ${serviceName + endpoint}Handler(w http.ResponseWriter, r *http.Request) {}\n\n"
```

Comment:
Is this clearer than $serviceName$endpoint?

Reply:
It's certainly no less clear